core/embed/trezorhal/secret.h

```diff
@@ -31,12 +31,14 @@ void secret_hide(void);
 
 void secret_write_header(void);
 
+secbool secret_optiga_set(uint8_t* secret);
```

Comment:
I assume that the set function does not modify the secret. We should use const in that case.

```suggestion
secbool secret_optiga_set(const uint8_t* secret);
```

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/86a5c1ac1f2ac8106e911db89b75ec029ffd874c